README.md

```diff
@@ -138,6 +138,17 @@ Restart your app server and try clicking on the “add important change” or 
 
 We need your help! Please read through the [Web Monitoring Project](https://github.com/edgi-govdata-archiving/web-monitoring) project document and see what you can help with and check [EDGI’s contribution guidelines](https://github.com/edgi-govdata-archiving/overview/blob/master/CONTRIBUTING.md) for information on how to propose issues or changes.
 
+## Docker
+
+The Dockerfile runs the rails server on port 3001 in the container. To build
+and run:
+
+```
+docker build -t ui . -e <ENVIRONMENT VARIABLES>
+docker run -p 3001:3001 ui
+```
+
+Point your browser or ``curl`` at ``http://localhost:3001``.
```

Comment:
Probably doesn’t make sense to mention `curl` here (it would for DB).

Reply:
I'm getting this error: `unknown shorthand flag: 'e' in -e` when trying to run the `docker build` command.

Reply:
Oh, yeah, that should be on the `docker run` command, shouldn’t it, @danielballan?